docs/transformation_grids.rst

```diff
@@ -0,0 +1,92 @@
+.. _transformation_grids:
+
+Transformation Grids
+=====================
+
+Transformation grids are necessary when you are performing datum transformations.
```

Comment:
They are not "necessary" (transformations also work if the grids are not available?), but make the transformation more accurate?

Reply:
That is true. I guess it depends on your definition of necessary.